nifi-extension-bundles/nifi-extension-utils/nifi-dbcp-base/src/main/java/org/apache/nifi/dbcp/utils/DBCPProperties.java

```diff
@@ -64,15 +66,31 @@ private DBCPProperties() {
             .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
             .build();
 
+    public static final PropertyDescriptor PASSWORD_SOURCE = new PropertyDescriptor.Builder()
+            .name("Password Source")
+            .description("Specifies whether to supply the database password directly or obtain it from a Database Password Provider.")
+            .allowableValues(PasswordSource.class)
+            .defaultValue(PasswordSource.PASSWORD.getValue())
```

Comment:
The `getValue()` call is not needed:
```suggestion
            .defaultValue(PasswordSource.PASSWORD)
```